plugins/netskope/help.md

```diff
@@ -0,0 +1,565 @@
+# Description
+
+This plugin allows users to create, read, update, and delete URL list
+
+# Key Features
+
+* Apple currently pending changes for URL lists
+* Create a new URL list
+* Delete specific URL list by ID
+* Get all applied or pending URL lists
+* Get single user confidence index
+* Get URL list by ID
+* Patch specific URL list by ID
+* Replace specific URL list by ID
+* Update the file hash list
+* Upload JSON config file
+
+
+# Requirements
+
+* Requires an API Keys for versions v1 and v2 from the product Netskope
```

Comment:
Are keys for BOTH versions 1 and 2 required? Or simply one or the other? How would the user know if they need to use API version 1 or 2?
  
```suggestion
* Requires an API Key for versions v1 and v2 from the Netskope product.
```

Reply:
Yes. Both API keys are required :smiley: